assets/src/blocks/Covers/CoversCarouselControls.js

```diff
@@ -0,0 +1,46 @@
+const { __ } = wp.i18n;
+
+const CoversCarouselArrow = ({ handler, direction, disabled = false }) => (
+  <button className={`carousel-control-${direction}`} disabled={disabled} onClick={handler}>
+    <span className={`carousel-control-${direction}-icon`} aria-hidden="true"><i></i></span>
+    <span className="visually-hidden">{__(direction === 'next' ? 'Next' : 'Prev', 'planet4-blocks')}</span>
+  </button>
+);
+
+export const CoversCarouselControls = ({
+  totalAmountOfCovers,
+  amountOfCoversPerRow,
+  slideCovers,
```

Comment:
As I mentioned in another comment, another approach could be to import the `useCovers` hook and `destructure the `slideCovers` method, instead of passing it as a prop. It's just a suggestion but it's up to you. 

And then [this](https://github.com/greenpeace/planet4-plugin-gutenberg-blocks/pull/734/files#r772401503)

Does it make sense?